flang/test/Lower/entry.f90

```diff
@@ -1,5 +1,19 @@
 ! RUN: bbc -emit-fir -o - %s | FileCheck %s
 
+program entries
+  character(10) hh, qq, m
+  integer mm
+  call ss(mm);     print*, mm
+  call e1(mm, 17); print*, mm
+  call e2(17, mm); print*, mm
+  call e3(mm);     print*, mm
+  print*, jj(11)
+  print*, rr(22)
+  m = 'abcd efgh'
+  print*, hh(m)
+  print*, qq(m)
+end
+
```

Comment:
Don't we want to test both directions? Calling before called and called before calling?

Reply:
The modifications to altret.f90 from the prior conversation should now test one direction, while this test covers the other direction.